.gitattributes

```diff
@@ -4,3 +4,4 @@ testdata/ filter=lfs diff=lfs merge=lfs -text
 testdata/bags filter=lfs diff=lfs merge=lfs -text
 testdata/bags/demo.bag filter=lfs diff=lfs merge=lfs -text
 testdata/mcap filter=lfs diff=lfs merge=lfs -text
+testdata/db3/chatter.db3 filter=lfs diff=lfs merge=lfs -text
```

Comment:
I think you can use a ** here to make anything in this folder gitlfs. Thats a safer bet than forgetting to add it to lfs.